widgets/package.json

```diff
@@ -1,10 +1,23 @@
 {
 	"name": "huggingface-widgets",
-	"version": "0.0.1",
+	"version": "0.15.0",
+	"type": "module",
+	"main": "./InferenceWidget/InferenceWidget.svelte",
```

Comment:
`main` is supposed to be something in plain js I think (that's why you can add a specific `svelte` entry for the "non-compiled" version), but for now, we can leave it as is - and add a "compiled" version later if there is a demand for it.